db/fileWriter.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+/* eslint-disable no-unused-vars */
+const faker = require('faker');
+const fs = require('fs');
+const pool = require('./pool.js');
+
+const startTime = Date.now();
+
+const writeObj = (index) => {
+  const newReview = {
+    product_id: Math.floor(Math.random() * 10),
+    review_id: index,
+    rating: faker.random.number(),
+    summary: faker.company.catchPhrase(),
+    recommend: faker.random.number(),
+    response: faker.company.catchPhrase(),
+    body: faker.company.catchPhrase(),
+    date: faker.date.recent(),
+    reviewer_name: faker.internet.userName(),
+    helpfulness: faker.random.number(),
+    photos: [faker.image.people(), faker.image.people()],
+    characteristics: {
+      Size: {
```

Comment:
My experience writing the reviews had each characteristic have their own id, even across products. so product 1 characteristics were 1,2,3 etc. while product 20 characteristics were 45,46,47 etc. (each characteristic is tied to the product, i.e. it's own table) so I'm wary to see the id's hard-coded.

If that's what works though, then totally ignore me.